app/repos/courts.js

```diff
@@ -13,6 +17,86 @@ const getCourts = async () => {
   }
 }
 
+/**
+ * @typedef CourtCreatePayload
+ * @property {string} name
+ */
+/**
+ * @param {CourtCreatePayload} courtData
+ * @param user
+ * @param [transaction]
+ * @returns {Promise<*|undefined>}
+ */
+const createCourt = async (courtData, user, transaction) => {
+  if (!transaction) {
+    return await sequelize.transaction(async (t) => createCourt(courtData, user, t))
+  }
+  const foundCourt = await sequelize.models.court.findOne({
+    where: {
+      name: courtData.name
```

Comment:
Does this automatically do a case-insensitive search? It does with SQL Server but think Postgres is case-sensitive

Reply:
If you use the 'like' and %% then it is case insensitive (see lookups methods)

Reply:
very good point